deployments/quick-start/vault_rbac.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-vault
+  namespace: default
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: kube-vault
+rules:
+- apiGroups: [""]
+  resources:
+  - pods
+  verbs: ["get", "list"]
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: prometheus-server-binding
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: kube-vault
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: kube-vault
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: vault
+  namespace: default
+  labels:
+    app: vault
+spec:
+  ports:
+    - name: vault
+      port: 8200
+  selector:
+    app: vault
+---
+apiVersion: extensions/v1beta1
```

Comment:
Should the roles and bindings be added to the deployment for kubernetes-vault instead?